src/screens/signup/otpVerification/index.tsx

```diff
@@ -4,10 +4,16 @@ import Header from '../../../commons/header/header';
 import {
   HeaderTitleContainer,
   HeaderTitleOf,
-  HeaderTitleNumber
+  HeaderTitleNumber,
+  CancelSetupButton
 } from './style';
+import { NavigationInterface } from '../../types';
```

Comment:
Move all imports to the top and leave style imports as last,
Like this should come before importing components from styles